subprojects/docs/src/docs/userguide/introductionDependencyManagement.adoc

```diff
@@ -15,33 +15,45 @@
-* _Integration with existing dependency management infrastructure_: Gradle is compatible with both Maven and Ivy repositories. If you use Archiva, Nexus, or Artifactory, Gradle is 100% compatible with all repository formats.
+=== Dependency management in Gradle
 
-With hundreds of thousands of interdependent open source components each with a range of versions and incompatibilities, dependency management has a habit of causing problems as builds grow in complexity. When a build's dependency tree becomes unwieldy, your build tool shouldn't force you to adopt a single, inflexible approach to dependency management. A proper build system has to be designed to be flexible, and Gradle can handle any situation.
+Gradle has built-in support for dependency management and lives up the task of fulfilling typical scenarios encountered in modern software projects. We’ll explore the main concepts with the help of an example project. The illustration below should give you an rough overview on all the moving parts.
 
++++++
+<figure xmlns:xi="http://www.w3.org/2001/XInclude">
+    <title>Dependency management big picture</title>
+    <imageobject>
+        <imagedata fileref="img/dependency-management-resolution.png" width="130mm" />
+    </imageobject>
+</figure>
++++++
 
-[[sub:dependency_management_and_migrations]]
-=== Flexible dependency management for migrations
+The example project builds Java source code. Some of the Java source files import classes from link:https://github.com/google/guava[Google Guava], a open-source library providing a wealth of utility functionality. In addition to Guava, the project needs the link:http://junit.org/junit5/[JUnit] libraries for compiling and executing test code.
 
-Dependency management can be particularly challenging during a migration from one build system to another. If you are migrating from a tool like Ant or Maven to Gradle, you may be faced with some difficult situations. For example, one common pattern is an Ant project with version-less jar files stored in the filesystem. Other build systems require a wholesale replacement of this approach before migrating. With Gradle, you can adapt your new build to any existing source of dependencies or dependency metadata. This makes incremental migration to Gradle much easier than the alternative. On most large projects, build migrations and any change to development process is incremental because most organizations can't afford to stop everything and migrate to a build tool's idea of dependency management.
+Guava and JUnit represent the _dependencies_ of this project. A build script developer can <<declaring_dependencies,declare dependencies>> for different scopes e.g. just for compilation of source code or for executing tests. In Gradle, the <<sub:scope_of_dependency_configurations,scope of a dependency>> is called a _configuration_. For a full overview, see the reference material on <<dependency_types,dependency types>>.
```

Comment:
This is very good. I only wish we had something to link to for "configuration".... we'll get there.

Reply:
> In Gradle, the <<sub:scope_of_dependency_configurations,scope of a dependency>> is called a _configuration_.

This sentence actually contains the link and it has the relevant information on configurations. In the previous PR I introduced subtitles for that section. Can you have a look at that again and let me know if that's what your are expecting? Maybe I am misunderstanding what you are saying here. Do you think we should link the _word_ "configuration" directly? Do you think linking "scope of dependency" is not obvious? We can definitely juggle with this a bit to optimize for searches. The actual information is there from my perspective.

Reply:
Optionally, we could link to the word "configuration" here.